src/sagemaker/processing.py

```diff
@@ -170,7 +174,9 @@ def _extend_processing_args(self, inputs, outputs, **kwargs):  # pylint: disable
         """Extend inputs and outputs based on extra parameters"""
         return inputs, outputs
 
-    def _normalize_args(self, job_name=None, arguments=None, inputs=None, outputs=None, code=None):
+    def _normalize_args(
+        self, job_name=None, arguments=None, inputs=None, kms_key=None, outputs=None, code=None
```

Comment:
can we make the `code` and `kms_key` as adjacent args, preferably after `code`, since they are the args that are associated?

```suggestion
        self, job_name=None, arguments=None, inputs=None, outputs=None, code=None, kms_key=None
```

Reply:
Yes, it makes more sense to put relevant args together.